src/measure_theory/constructions/polish.lean

```diff
@@ -0,0 +1,686 @@
+/-
+Copyright (c) 2022 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import topology.metric_space.polish
+import measure_theory.constructions.borel_space
+
+/-!
+# The Borel sigma-algebra on Polish spaces
+
+We discuss several results pertaining to the relationship between the topology and the Borel
+structure on Polish spaces.
+
+## Main definitions and results
+
+First, we define the class of analytic sets and establish its basic properties.
+
+* `analytic_set s`: a set in a topological space is analytic if it is the continuous image of
+  a Polish space. Equivalently, it is empty, or the image of `ℕ → ℕ`.
+* `analytic_set.image_of_continuous`: a continuous image of an analytic set is analytic.
```

Comment:
```suggestion
* `measure_theory.analytic_set s`: a set in a topological space is analytic if it is the continuous image of
  a Polish space. Equivalently, it is empty, or the image of `ℕ → ℕ`.
* `measure_theory.analytic_set.image_of_continuous`: a continuous image of an analytic set is analytic.
```

Reply:
I assume that these analytic sets have nothing to do with the analytic sets in analytic geometry, right? (I.e. zero sets of analytic functions.)

Edit: I now see that you answer this in the docstring :smile: